sources/kernel/syscalls.c

```diff
@@ -194,6 +195,18 @@ BrResult sys_create_space(BrSpace *handle, BrCreateSpaceArgs *args)
     return BR_SUCCESS;
 }
 
+BrResult sys_create_irq(BrIrq *handle, BrCreateIrqArgs *args)
+{
+    auto irq = irq_create(args->irq);
```

Comment:
Instead of creating a new IrqObject every time we could reuse another IrqObject with the same IRQ number.

Reply:
but what happend if multiples task map an irq ?

Reply:
Not a problem we do recounting